econml/tests/test_drlearner.py

```diff
@@ -0,0 +1,558 @@
+                                                                      [2, 2 + X.shape[1] + (W.shape[1] if W is not None else 0)])
+                                        np.testing.assert_array_equal(np.array([mdl.feature_importances_ for mdl in est.models_propensity]).shape,
+                                                                      [2, X.shape[1] + (W.shape[1] if W is not None else 0)])
+                                    else:
+                                        np.testing.assert_array_equal(np.array([mdl.coef_ for mdl in est.models_regression]).shape,
+                                                                      [2, 2 + X.shape[1] + (W.shape[1] if W is not None else 0)])
+                                        np.testing.assert_array_equal(np.array([mdl.coef_ for mdl in est.models_propensity]).shape,
+                                                                      [2, 3, X.shape[1] + (W.shape[1] if W is not None else 0)])
+                                    if multitask_model_final:
+                                        if isinstance(models[2], RandomForestRegressor):
+                                            np.testing.assert_equal(np.argsort(
+                                                est.multitask_model_cate.feature_importances_)[-1], 0)
+                                        else:
+                                            true_coef = np.zeros((2, len(out_feat_names)))
+                                            true_coef[:, 0] = [.5, 1]
+                                            np.testing.assert_allclose(
+                                                est.multitask_model_cate.coef_, true_coef, rtol=0, atol=.15)
+                                            np.testing.assert_allclose(
+                                                est.multitask_model_cate.intercept_, [1, 2], rtol=0, atol=.15)
+                                    else:
+                                        for t in [1, 2]:
+                                            if isinstance(models[2], RandomForestRegressor):
+                                                np.testing.assert_equal(np.argsort(
+                                                    est.model_cate(T=t).feature_importances_)[-1], 0)
+                                            else:
+                                                true_coef = np.zeros(len(out_feat_names))
+                                                true_coef[0] = .5 * t
+                                                np.testing.assert_allclose(
+                                                    est.model_cate(T=t).coef_, true_coef, rtol=0, atol=.15)
+                                                np.testing.assert_allclose(
+                                                    est.model_cate(T=t).intercept_, t, rtol=0, atol=.15)
+
+    def test_linear_drlearner_all_attributes(self):

```

Comment:
Is there a reason to have separate tests for DRLearner vs. LinearDRLearner?  Looks like a lot of duplicated code.

Reply:
Yeap. The two have different input params and diffeerent attributes after fitting.

Reply:
Also the drlearner has no interval capabilities

Reply:
Okay.  At some point I'd like to consolidate the repeated logic but that can be done later.